### PR TITLE
Fixes 10472

### PR DESCRIPTION
**Pull request by coffmant**
Fixes 10472: TCP port for qdrouterd needs to be added to the docs

### CODE REVIEW

File: docs/installation/index.md [hunk 1]
```diff
@@ -35,6 +35,7 @@ The following ports need to be open to external connections:
 
 * 80 TCP - HTTP, used for provisioning purposes
 * 443 TCP - HTTPS, used for web access and api communication
+* 5647 TCP - qdrouterd - used for client and capsule actions
 * 5671 TCP - qpid ssl - used for client actions
```
Comment (stbenjam) [inline docs/installation/index.md]: Would you mind removing this line, too? I don't think we need this one anymore for external connections, everything happens through the dispatch router.

**Comment by stbenjam** [comment]
@coffmant Thanks! Just a small comment (and please [squash](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html) into one commit).

**Comment by coffmant** [comment]
Removed the other line and squashed the commit

**Comment by stbenjam** [comment]
@coffmant Looks like the wrong port, 9090 can stay, 5671 (qpidd) should go

**Comment by coffmant** [comment]
@stbenjam Updated to fix the port mixup.  Thanks

**Comment by stbenjam** [comment]
@coffmant Thanks! Looks good, will merge once tests are green (it's an unrelated issue with some broken links elsewhere)

**Comment by stbenjam** [comment]
[test]

**Comment by stbenjam** [comment]
Thanks @coffmant, merged as a6660b9be189fcc0084dcae14f0a3676bffe7d81.

**Pull request closed without merging**